File: verification-packs/polyglot-30-route-formal-equivalence-v1/evidence/routes/rust-to-typescript/certification/artifacts/holdout/source-runtime/source_harness.rs
include!("clamp.rs");

fn main() {
    let actual_0 = clamp(20, 10);
    assert!(actual_0 == 10, "case 0");
    println!("ELMOS_OBSERVATION\t0\trust-debug\t{:?}", actual_0);
    let actual_1 = clamp(-2, 10);
    assert!(actual_1 == 0, "case 1");
    println!("ELMOS_OBSERVATION\t1\trust-debug\t{:?}", actual_1);
    let actual_2 = clamp(7, 10);
    assert!(actual_2 == 7, "case 2");
    println!("ELMOS_OBSERVATION\t2\trust-debug\t{:?}", actual_2);
}

File: verification-packs/polyglot-30-route-formal-equivalence-v1/evidence/routes/python-to-typescript/certification/artifacts/holdout/source-runtime/source_harness.py
import json
import clamp
actual_0 = clamp.clamp(20, 10)
assert actual_0 == 10
print("ELMOS_OBSERVATION\tjson\t" + json.dumps({"case_id": 0, "value": actual_0}, sort_keys=True, separators=(",", ":")))
actual_1 = clamp.clamp(-2, 10)
assert actual_1 == 0
print("ELMOS_OBSERVATION\tjson\t" + json.dumps({"case_id": 1, "value": actual_1}, sort_keys=True, separators=(",", ":")))
actual_2 = clamp.clamp(7, 10)
assert actual_2 == 7
print("ELMOS_OBSERVATION\tjson\t" + json.dumps({"case_id": 2, "value": actual_2}, sort_keys=True, separators=(",", ":")))
